src/hotspot/share/classfile/classLoaderExt.cpp

```diff
@@ -310,7 +295,7 @@ struct CachedClassPathEntry {
 
 static GrowableArray<CachedClassPathEntry>* cached_path_entries = NULL;
 
-ClassPathEntry* ClassLoaderExt::find_classpath_entry_from_cache(const char* path, TRAPS) {
+ClassPathEntry* ClassLoaderExt::find_classpath_entry_from_cache(Thread* current, const char* path) {
```

Comment:
If this is executed by the vmThread at dump time then I'd prefer to  see current called vmThread.

Reply:
This is executed only in the main Java thread.